src/components/Indicator/Indicator.tsx

```diff
@@ -2,14 +2,20 @@ import * as React from 'react';
 import {classNames} from '@shopify/css-utilities';
 import styles from './Indicator.scss';
 
+type Tone = 'positive' | 'neutral' | 'negative';
+
```

Comment:
Wondering if `Tone` is a better choice than `Status`. 

Banners have:
`Status = 'success' | 'info' | 'warning' | 'critical'`

Badge has: 
`Status = 'success` | `info` | `attention` | `warning` | `new`

This could be 
`Status = 'neutral' | 'info' | 'warning' | 'critical'`

cc: @sarahill

Reply:
Agree with using status vs. tone. Keeps the same mental model.